docs/testing/debug-add-ins-using-f12-tools-ie.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Debug add-ins using developer tools for Internet Explorer
+description: 'Debug add-ins using the developer tools in Internet Explorer'
+ms.date: 10/20/2021
+ms.localizationpriority: medium
+---
+
+# Debug add-ins using developer tools in Internet Explorer
+
+This article shows how to debug the client-side code (JavaScript or TypeScript) of your add-in when the following conditions are met:
+
+- You cannot, or don't wish to, debug using tools built into your IDE; or you are encountering a problem that only occurs when the add-in is run outside the IDE.
+- Your computer is using a combination of Windows and Office versions that use the Internet Explorer webview control, Trident.
+
+To determine which browser is being used on your computer, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```

Comment:
```suggestion
To determine which browser you're using, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```

Reply:
See other comment. This edit would change the meaning.